src/services/fileStorage/proxy-service.js

```diff
@@ -67,28 +59,28 @@ const prepareThumbnailGeneration = (file, strategy, userId, { name: dataName },
 			}),
 			strategy.generateSignedUrl({
 				userId,
-				flatFileName: storageFileName.replace(
-					/(\..+)$/,
-					'-thumbnail.png',
-				),
+				flatFileName: storageFileName.replace(/(\..+)$/, '-thumbnail.png'),
 				fileType: returnFileType(dataName || propName), // data.type
 			}),
-		]).then(([downloadUrl, signedS3Url]) => rp.post({
-			url: FILE_PREVIEW_SERVICE_URI,
-			body: {
-				downloadUrl,
-				signedS3Url,
-				callbackUrl: url.resolve(FILE_PREVIEW_CALLBACK_URI, file.thumbnailRequestToken),
-				options: {
-					width: 120,
-				},
-			},
-			json: true,
-		}).catch((err) => {
-			logger.warning(new Error('Can not create tumbnail', err)); // todo err message is lost and throw error
-		}))
-		: Promise.resolve()
-);
+		  ]).then(([downloadUrl, signedS3Url]) =>
+			rp
+				.post({
+					url: FILE_PREVIEW_SERVICE_URI,
+					body: {
+						downloadUrl,
+						signedS3Url,
+						callbackUrl: url.resolve(FILE_PREVIEW_CALLBACK_URI, file.thumbnailRequestToken),
+						options: {
+							width: 120,
+						},
+					},
+					json: true,
+				})
+				.catch((err) => {
+					logger.warning(new Error('Can not create tumbnail', err)); // todo err message is lost and throw error
```

Comment:
Codacy found an issue: [Replace `logger.warning(new·Error('Can·not·create·tumbnail',·err)` with `↹↹↹logger.warning(⏎↹↹↹↹↹↹↹new·Error('Can·not·create·tumbnail',·err),⏎↹↹↹↹↹↹`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)